oop_verify.py

```diff
@@ -42,6 +42,138 @@ def get_content(self):
         return soup
 
 
```

Comment:
In Python, if you want to inherit a super class without changing the behavior, you don't have to override the superclass's method and  "return Superclass.method()". Simply init the subclass by "super.__init__(*args, **kwargs)".